pkg/server/plugin/upstreamauthority/awspca/pca_test.go

```diff
@@ -186,37 +186,45 @@ func (as *PCAPluginSuite) Test_MintX509CA() {
 	as.Require().NoError(err)
 	as.verifyGetCertificate(encodedCert, encodedCertChain, nil)
 
-	// The resulting response should not error, and should contain the expected
-	// values from ACM.
-	response, err := as.plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{
+	stream, err := as.plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{
 		Csr:          csr,
 		PreferredTtl: testTTL,
 	})
 	as.Require().NoError(err)
+	as.Require().NotNil(stream)
+
+	// The resulting response should not error, and should contain the expected
+	// values from ACM.
+	response, err := stream.Recv()
+	as.Require().NoError(err)
 	as.Require().NotNil(response)
 	as.Require().Equal([][]byte{expectedCert.Raw, expectedIntermediate.Raw}, response.X509CaChain)
 	as.Require().Equal([][]byte{expectedRoot.Raw}, response.UpstreamX509Roots)
 }
 
 func (as *PCAPluginSuite) Test_MintX509CA_IssuanceError() {
-	defer as.mockController.Finish()
+	as.pcaClientFake.recycle(as.T())
 	as.configurePlugin()
 
 	// Issuance returns an error
 	csr, expectedEncodedCsr := as.generateCSR()
 	as.verifyIssueCertificate(expectedEncodedCsr, errors.New("issuance error"))
 
 	// The resulting response should return an error
-	response, err := as.plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{
+	stream, err := as.plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{
 		Csr:          csr,
 		PreferredTtl: testTTL,
 	})
+	as.Require().NoError(err)
+	as.Require().NotNil(stream)
+
+	response, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns `io.EOF`.